source/change-streams/change-streams.rst

```diff
@@ -165,10 +208,16 @@ Driver API
 
   interface ChangeStream extends Iterable<Document> {
     /**
-     * The resume token (_id) of the document the iterator last returned
+     * The token to be used during the resume process.
      */
     private resumeToken: Document;
 
+    /**
+     * A public getter for resumeToken. If resumeToken has not been set yet,
+     * returns a disengaged optional.
```

Comment:
According to a Google search, "disengaged optional" seems like a C++ term that implies that optional cannot be dereferenced. Is that correct?

Reply:
agreed, I think you can let the TypeScript stand for itself here, `A public getter for resumeToken` is sufficient

Reply:
"disengaged optional" sounds cool but I concede.

Reply:
Sorry, I wasn't suggesting its removal -- just asking a clarifying question.